burstphoto/align.metal

```diff
@@ -209,6 +485,84 @@ kernel void compute_tile_alignments(texture3d<float, access::read> tile_diff [[t
 }
 
 
+// see section on "Hierarchical alignment" in https://graphics.stanford.edu/papers/hdrp/hasinoff-hdrplus-sigasia16.pdf and section "Multi-scale Pyramid Alignment" in https://www.ipol.im/pub/art/2021/336/
```

Comment:
I think that you should include a description of how this function works here rather than pointing the person to the papers.

Reply:
A short description can be added in a future version of the app.